### PR TITLE
Set Min Width of Navigator

### LINKED ISSUE #47: Horizontal Scrollbar disappears
Scrollbar disappear

### PR DESCRIPTION
**Pull request by muddana-satish**
fixes #47 

Signed-off-by: muddana-satish <satish.muddana@ericsson.com>

### CODE REVIEW

File: timeline-chart/src/layer/time-graph-navigator.ts [hunk 1]
```diff
@@ -80,7 +80,8 @@ export class TimeGraphNavigatorHandle extends TimeGraphComponent {
 
     render(): void {
         const position = { x: this.unitController.viewRange.start * this.stateController.absoluteResolution, y: 0 };
-        const width = this.unitController.viewRangeLength * this.stateController.absoluteResolution;
+        // Set min width to 20 to make it appear at all zoom levels
+        const width = Math.max(20, this.unitController.viewRangeLength * this.stateController.absoluteResolution);
```
Comment (mirsky-work) [inline timeline-chart/src/layer/time-graph-navigator.ts:84]: Perhaps extract the value `20` into a constant, and then the comment isn't required?

**Review by mirsky-work — COMMENTED** [review]
Validated and it works well within theia-trace-extension. Initially experienced some quirks on a VM without WebGL 2, but along with PR #48 it was smooth.

File: timeline-chart/src/layer/time-graph-navigator.ts (hunk 1 above)
Comment (muddana-satish) [reply timeline-chart/src/layer/time-graph-navigator.ts]: updated.

**Review by tahini — APPROVED** [review]
Looks good. Probably un-related bug I noticed while testing this one. If you move the bar completely to the right and zoom in, the little scrollbar goes to hide under the black gap like in the screenshot below.

hiddenScrollbar

**Review by MatthewKhouzam — APPROVED** [review]

**Pull request merged**